src/sherwood.cr

```diff
@@ -95,6 +95,15 @@ class Sherwood
 
       # TODO: Variable Operations
       # TODO: Control Flow
+      when 0x70 then insp -= popType(SWInt, stack)
+      when 0x71 then
+        if popType(Bool, stack) then {
```

Comment:
Your code failing checks wasn't from renaming `reference.txt`, but rather from this. Crystal's `if` syntax is this:
```crystal
if cond
  # stuff
else
  # stuff
end
```
Not this:
```crystal
if cond {
  # stuff
} else {
  # stuff
}
```

Reply:
Idk how Crystal works so sorry, I can't. Ill have to rely on travis, but thats for the notice.

Reply:
It's just a command line call to the compiler. The `crystal spec` command from the root directory of the project should run the tests without any fuss.